src/odemis/acq/stream_settings.py

```diff
@@ -0,0 +1,174 @@
+# -*- coding: utf-8 -*-
+"""
+Created on 24 Jan 2024
+
+@author: Karishma Kumar
+
+Copyright © 2024 Karishma Kumar, Delmic
+
+This file is part of Odemis.
+
+Odemis is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License version 2 as published by the Free Software Foundation.
+
+Odemis is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with Odemis. If not, see http://www.gnu.org/licenses/.
+"""
+import json
+import logging
+import os
+from typing import List, Dict, Any
+
+from odemis import model, util
+from odemis.acq.stream import FluoStream
+from odemis.util.img import md_format_to_tint, tint_to_md_format
+
+SETTINGS_ORDER = ["name", "excitation", "power", "emission", "tint", "detExposureTime", "auto_bc_outliers"]
```

Comment:
Please add a comment explaining why the order matters. 
Actually, I thought the order is just to handle the fact that changing the excitation & emission can affect the tint. So only these 3 should be listed. Or do the name also has an effect? (I think it's just a GUI thing, so changing the name directly on the stream wouldn't have any effect.